### ISSUE TITLE
Accessibility: Fix MySQL config page fastpass issues

### ISSUE THREAD

**Issue by kaydelaney** [grafana/grafana#42719]

**Comment by codeincarnate** [comment]
Closing as this should be fixed with #43982
